### DISCUSSION TITLE
Privacy policy requirements for a website using D3

### DISCUSSION THREAD

**Discussion by tvercaut** [d3/d3#3842] [Q&A] [answered]
I would like to create a webpage that uses D3 (more specifically the [d3-dsv](https://d3js.org/d3-dsv) part) and am wondering what I need to add to my website privacy policy with respect to D3 specifically.

I tried using [iubenda](https://www.iubenda.com/) for it but D3 is not a recognised service there. Any help would be appreciated.

For context, here is what I currently use as an example to allow using google fonts:

> **Displaying content from external platforms**
> _Google Fonts (Google Ireland Limited)_
>
> Google Fonts is a typeface visualization service provided by Google Ireland Limited that allows this Application to incorporate content of this kind on its pages.
>
> Personal Data processed: Usage Data; various types of Data as specified in the privacy policy of the service.
>
> Place of processing: Ireland – [Privacy Policy](https://policies.google.com/privacy).

Is there an authoritative link I can point to confirming that D3 does not log or collect any personal information?

**Comment by curran** [comment, marked as answer]
Perhaps you could link to the source code?

https://github.com/d3/d3-dsv/tree/main/src

**Reply by tvercaut** [reply]
Thank you for the suggestion. I was hoping to be able to link to a statement from D3 that it does not collect any personal information. Something along the lines of what CreateJS has maybe:
https://createjs.com/legal/privacy.html
